src/auro-accordion.js

```diff
@@ -56,7 +57,12 @@ class AuroAccordion extends LitElement {
         reflect: true
       },
       noanimation: {
-        type: Boolean
+        type: Boolean,
+        reflect: true
+      },
+      showOverflow: {
+        type: Boolean,
+        reflect: true
```

Comment:
The article was inspired by the references we outlined here. http://auro.alaskaair.com/generator/best-practice

But I am not sure that your assumptions are correct. The concept of the reflection is that when something is a CSS hook AND has a functional component, it needs to be reflected as illustrated in this PR https://github.com/AlaskaAirlines/auro-loader/blob/main/src/auro-loader.js#L64-L79

Was this tested in your app and did it work?

Reply:
Using noanimation, I have to add a Svelte action in order for it to actually add the attribute and function as expected. Simply setting it on the accordion doesn't work in our svelte app. Per the article:
![image](https://user-images.githubusercontent.com/25255197/142512708-835e495e-57ff-4522-8b3b-d5e8b437072c.png)
I took this to mean that since we don't know what frontend framework our consumers are going to be using, we want to have reflect to make sure that either works. Maybe I'm misunderstanding?

Reply:
@geoffrich can you throw an assist in here?

Reply:
In this case, I'd recommend removing the property entirely, since we don't need to access it in JS. The `:host` styling hook will be enough to detect if the consumer has set the attribute.

If we did need to access this property in JS, then we _would_ need to reflect if we also wanted the attribute to be a styling hook. But in this case, it's easier to just remove the property. Svelte will set the attribute automatically because there's no corresponding property.